conf/modules.config

```diff
@@ -252,29 +288,42 @@ process {
         ]
     }
 
-    withName: BOWTIE2_ASSEMBLY_ALIGN {
-        ext.args = params.bowtie2_mode ? params.bowtie2_mode : params.ancient_dna ? '--very-sensitive -N 1' : ''
+    withName: GENOMAD_ENDTOEND {
+        ext.args = [
+            "--cleanup",
+            "--min-score ${params.genomad_min_score}",
+            "--splits ${params.genomad_splits}",
+        ].join(' ').trim()
         publishDir = [
-            path: { "${params.outdir}/Assembly/${assembly_meta.assembler}/QC/${assembly_meta.id}" },
+            path: { "${params.outdir}/VirusIdentification/geNomad/${meta.id}" },
             mode: params.publish_dir_mode,
-            pattern: "*.log"
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
 
-    withName: 'MAG_DEPTHS_PLOT|MAG_DEPTHS_SUMMARY|MAG_DEPTHS_PLOT_REFINED' {
+    withName: BOWTIE2_ASSEMBLY_ALIGN {
+        ext.args = params.bowtie2_mode ? params.bowtie2_mode : params.ancient_dna ? '--very-sensitive -N 1' : ''
+        ext.prefix = { "${meta.id}.assembly" }
         publishDir = [
-            path: { "${params.outdir}/GenomeBinning/depths/bins" },
-            mode: params.publish_dir_mode,
-            pattern: "*.{png,tsv}"
+            [
+                path: { "${params.outdir}/Assembly/${assembly_meta.assembler}/QC/${assembly_meta.id}" },
+                mode: params.publish_dir_mode,
+                pattern: "*.log"
+            ],
+            [
+                path: { "${params.outdir}/Assembly/${assembly_meta.assembler}/QC/${assembly_meta.id}" },
+                mode: params.publish_dir_mode,
+                pattern: "*.{bam,bai}",
+                enabled: params.save_assembly_mapped_reads
+            ],
```

Comment:
Didn't know we could pass two or more rules to publishDir like this. Pretty cool!

Reply:
Nifty isn't it!